morrf_init init
---
multi_objective_path[] paths
